src/ucp/rma/rma.inl

```diff
@@ -168,8 +168,10 @@ ucp_ep_rma_handle_fence(ucp_ep_h ep, ucp_request_t *req,
 
     /* Apply a fence if EP's sequence is behind worker's */
     if (ucs_unlikely(req->flags & UCP_REQUEST_FLAG_FENCE_REQUIRED)) {
-        if (ucs_likely(
-            ucs_is_pow2_or_zero(ep->ext->unflushed_lanes | lane_map))) {
+        if (ucs_unlikely(ep->ext->unflushed_lanes == 0)) {
```

Comment:
seems it is indeed unlikely case, what is the test which caught this?

Reply:
it was found in MTT